gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,32 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm_svc = PbmService.new(server, username, password) if @apiVersion >= '5.5'
```

Comment:
Does this create a stateful session the same way VIM does? If so, does it need to be explicitly closed the way VIM needs to be?